apps/web/src/services/administration/iam/user/modules/user-management-modal/modules/AdminRole.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+    <div class="admin-role-wrapper">
+        <div class="admin-role-headline">
+            <p-toggle-button
+                :value="state.isToggled"
+                @change-toggle="handleUpdateToggle"
+            />
+            <span class="title">{{ $t('IDENTITY.USER.FORM.ASSIGN_DOMAIN_ROLE') }}</span>
+        </div>
+        <!-- CAUTION: Do not remove key binding at select dropdown. This is for initiating scroll parent to refresh fixed menu style. -->
+        <p-select-dropdown v-if="state.isToggled"
+                           :key="props.activeTab"
```

Comment:
suggestion: I recommend more detailed key.
```suggestion
                           :key=`admin-role-${props.activeTab}`
```